src/west/app/project.py

```diff
@@ -865,10 +865,15 @@ def _update_concurrent_thread(args):
             saved_stdout = os.dup(sys.stdout.fileno())
             saved_stderr = os.dup(sys.stderr.fileno())
 
+            logfile = os.path.join(logdir, project.name)
+
+            # the name might contain `/`
+            os.makedirs(os.path.dirname(logfile), exist_ok=True)
```

Comment:
There is a newer  https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir . I believe `pathlib` is preferred in general for various reasons and `west` already uses it.

According to `setup.py` west requires python>=3.6